protocol/requests/share_image_message.go

```diff
@@ -0,0 +1,28 @@
+package requests
+
+import (
+	"errors"
+
+	"github.com/status-im/status-go/eth-node/types"
+)
+
+var ErrShareMessageInvalidID = errors.New("share image message: invalid id")
+var ErrShareMessageEmptyUsers = errors.New("share image message: empty users")
+
+type ShareImageMessage struct {
+	MessageID   string           `json:"id"`
+	Users       []types.HexBytes `json:"users"`
+	ContentType int32            `json:"contentType"`
```

Comment:
Do you need to pass `ContentType` from the client? Is it not always an image?

Reply:
yeah i actually don't have to